src/Databases/PostgreSQL/DatabasePostgreSQL.h

```diff
@@ -79,6 +84,9 @@ class DatabasePostgreSQL final : public IDatabase, WithContext
     std::shared_ptr<IDisk> db_disk;
     LoggerPtr log;
 
+    bool persistent = true;
```

Comment:
What does it mean if the database is not persistent?

Reply:
For some weird reason we have a possibility to detach/drop tables inside MySQL and PostgreSQL databases so that they aren't actually dropped but simply ignored on the ClickHouse side. To do it we create a persistent directory on disk where we store the list with ignored tables.

With this flag set to false we no longer support this feature and the database does not store anything on disk.

Reply:
According to the code `persistent` is always true, you set it to `true` and never set it to `false`.

Reply:
Yes, it is set to false in private repo based on a condition

Reply:
So if a database is not `persistent` - what exactly does that mean? You said we could mark some tables to be ignored, but this flag is for a database, and not for a table. Also I think a comment in the code could be helpful.

Reply:
When `persistent = false,` the database no longer stores anything on the disk and, therefore, no longer allows you to ignore tables inside of it.